10-JavaScript/ajax-intro.md

```diff
@@ -113,11 +113,13 @@ $.get(url, successCallback);
 Oftentimes, you'll see the above collapsed into one function call:
 ```javascript
 $.get('https://petdibs.herokuapp.com/pets',
-  response => {
+  (response) => {
     console.log('success!');
   });
 ```
 
+Notice our use of arrow functions here.  Because of their compact syntax and the fact that an arrow function gets `this` from it's outside container, they often make great callback functions.  
```

Comment:
Arguably the more important of the two benefits for arrows functions as callbacks is the way it handles `this`. If possible I think it would be helpful for the examples above to involve some use of `this`.

Reply:
Additionally, the code sample below (in the "Let's add callbacks..." section) should be updated to use the same arrow syntax as we have in the sample above.

Reply:
Seconded, maybe access/utilize the `response` obj